modules/auxiliary/scanner/http/titan_ftp_admin_pwd.rb

```diff
@@ -89,15 +89,39 @@ def run_host(ip)
         print_good("#{ip}:#{datastore['RPORT']} - Base Directory: #{info[:basedir]}")
       end
       print_good("#{ip}:#{datastore['RPORT']} - Admin Credentials: '#{info[:username]}:#{info[:password]}'")
-      report_auth_info(
-        :host       => ip,
-        :port       => datastore['RPORT'],
-        :user       => info[:username],
-        :pass       => info[:password],
-        :ptype      => "password",
-        :proto      => "http",
-        :sname      => "Titan FTP Admin Console"
+      report_cred(
+        ip: ip,
+        port: 21,
```

Comment:
@wchen-r7 If you still have the vulnerable FTP version available, can you please check if the XML-RPC response include the FTP port as well? Most of the time it will be the default but in some cases the user might have a different FTP port for which the credentials can be leveraged.